cloud/src/meta-store/keys.h

```diff
@@ -299,6 +417,130 @@ std::string system_meta_service_encryption_key_info_key();
 // clang-format on
 // TODO: add a family of decoding functions if needed
 
+namespace versioned {
+
+// ATTN: if the versionstamp is NOT provided, the Versionstamp::min() will be used.
+
+// clang-format off
+void partition_version_key_prefix(const PartitionVersionKeyInfo& in, std::string* out);
```

Comment:
the output param form is unnecessary, the form will be refactored (soon)
we keep the return value one is ok.